app/views/devise/registrations/edit.html.erb

```diff
@@ -7,6 +7,7 @@
     <%= f.input :name, label: "Имя", autofocus: true, required: false %>
     <%= f.input :surname, label: "Фамилия", required: false %>
 
+    <%= f.input :submission_form_type, collection: { "Ctrl+Enter" => :ctrl_enter, "Shift+Enter" => :shift_enter, "Enter" => :enter }, label: "Сочетания клавиш для отправки формы" %>
```

Comment:
How would an already-registered user know there're now such an option here? Why not add it to a place where it's actually needed?
<img width="301" alt="screen shot 2016-11-29 at 12 47 33" src="https://cloud.githubusercontent.com/assets/3923587/20704997/53ebe226-b632-11e6-9576-de984918bac0.png">

Do you really want every user to choose this even before they used the site when they're signing up?